README.md

```diff
@@ -7,7 +7,7 @@
 [![Support badge]( https://img.shields.io/badge/support-sof-yellowgreen.svg)](http://stackoverflow.com/questions/tagged/orchestrator/)
 [![Join the chat at https://gitter.im/telefonicaid/orchestrator](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/telefonicaid/orchestrator?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Orchestrator tries to group all provision operations for IoT platform that tipically implies several steps or several systems interaction.
+Orchestrator tries to group all provision operations for IoT platform that typically implies several steps or several systems interactions.
```

Comment:
I'm not an expert in English grammar :), but I'll say that either "system interactions" or "systems interaction" are ok but that the usage of both words in plural ("systems interactions") isn't... What do you think?